brainstormforce/ultimate-addons-for-gutenberg
# Fix: Advanced Columns - Margin option missing in spacing setting for inner column.

### Description

- Trello - https://trello.com/c/LDeRdctr/308-fix-advanced-columns-margins-setting-missing-in-spacing-for-an-inner-column

### Screenshots

- Issue - https://www.loom.com/share/c680e2f59d3d40d1af6dca9ca70da651

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the Advanced column.
- Check inner column > spacing settings > Margin option.

### Checklist:
